wire-ios/Wire-iOS/Sources/UserInterface/Components/Views/SectionHeader.swift

```diff
@@ -22,14 +22,15 @@ import WireDesign
 
 final class SectionHeaderView: UIView {
 
-    let titleLabel = DynamicFontLabel(style: .h5,
-                                      color: SemanticColors.Label.textSectionHeader)
+    let titleLabel = DynamicFontLabel(
+        style: .h5,
+        color: SemanticColors.Label.textSectionHeader
```

Comment:
Shouldn't we use colors from ColorTheme?

Reply:
I haven’t had enough time to fully explore the ColorTheme and how we define new colors. Currently, we have two systems in place: SemanticColors and ColorTheme. My plan is to sync with the design team to streamline our color usage across both design files and code. For now, I’ll leave the implementation as is.

Thank you for the reminder; I definitely need to investigate this further.